ukrainian/normal/normal.md

```diff
@@ -0,0 +1,3 @@
+﻿Ви починаєте займатися своїми звичайними справами адже у вашому зефірному світі нічого не змінилося...
+
+Повернутися до початку можна [тут] (marshmallow.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.